src/app/screens/Dashboard/screens/Home/components/CardsContainer/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { withSpinner } from '~components/Spinner';
+
+import Card from '../Card';
+import styles from './styles.module.scss';
+
+interface Props {
+  components: {
+    title: string,
+    html: string,
+    css: string
+  }[],
+  category: string
+}
+
+function CardsContainer({ components, category }: Props) {
+  return (
+    <div className={styles.cardContainer}>
+      {components &&
+        components.map((comp, index) => (
```

Comment:
I think we have optional chaining in our dependencies, don't we ? 

```suggestion
      {components?.map((comp, index) => (
```

Though, you could always get an empty array so it's a good idea to check that

```suggestion
      {!!components?.length &&
        components.map((comp, index) => (
```